docs/user/apirule-migration/01-84-migrate-oauth2-v1beta1-to-v2.md

```diff
@@ -210,6 +215,10 @@ The following APIRule example delegates token validation to the previously confi
 
     For preflight requests to work correctly, you must explicitly add the `"OPTIONS"` method to the **rules.methods** field of your APIRule `v2`. For guidance, see the [APIRule `v2` examples](../custom-resources/apirule/04-10-apirule-custom-resource.md#sample-custom-resource).
 
+> [!WARNING]
+> If you had a case with multiple APIRules targeting the same workload and you applied an additional, temporary AuthorizationPolicy to allow traffic for workloads exposed by APIRules `v1beta1` during the migration. Remember to delete this AuthorizationPolicy, please refer to the last point in procedure: [Migrate multiple APIRules `v1beta1` targeting same workload](./01-90-migrate-multiple-apirules-targeting-same-workload.md).
+>
+
```

Comment:
```suggestion
> [!WARNING]
> If you migrated multiple APIRules that target the same workload, and you applied an additional AuthorizationPolicy to avoid traffic disruption during migration, delete it. For instructions, see the last point of the procedure [Migrate multiple APIRules `v1beta1` targeting same workload](./01-90-migrate-multiple-apirules-targeting-same-workload.md).
```